src/CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+    * For example: `01-orientation.md`.
+* When referring to filenames, be sure to specify the relatiive path to file. This is to make it easy to copy-paste the filename when people use the tutorial.
+    * For example: *"Let's start by creating a new file at `app/components/jumbo.hbs`."*
+* When referring to other chapters, link to the chapter using its relative path in the tutorial.
+    * For example: `We learned about this in a [previous chapter](../02-building-pages/)`.
+* When referring to important key terms, be sure to *italicize* the term and link to a definition for it.
+    * For example: `*[route](http://a-great-link-to-the-definition-of-a-route-goes-here)*`.
+* When referring to component names, keywords, helpers, or HTML elements, be sure to use code markup.
+    * For example: *"Inside of this file, we have the `<NavBar>` component and the `{{outlet}}` keyword."*
+* When using the "Zoey says" callouts, be sure to use the blockquote formatting, ensuring that an extra `>` is included between the "Zoey says..." text and the actual note content. This is required in order to properly convert the markdown into HTML.
+    * For example:
+        ```
+        > Zoey says...
+        >
+        > Here's a very helpful comment!
+        ```
+
+## Making changes to the tutorial code
+
+The workflow for updating the Super Rentals Tutorial involves working within two directories of this project: `/src/chapters` and `/dist`. The editable prose lives in `src/chapters`; any changes that are made to the prose should be done there. The tutorial uses the content in `/src/chapters` to generate the prose markdown that is output into `/dist/chapters`. Interspersed with the prose are commands that are used to generate the code for the _actual_ tutorial app, which is outputted at `/dist/code/super-rentals`.
+
+Once you have made your changes in the tutorial prose or code, you can run a few different commands to verify that your changes render as expected.
+
+* To regenerate all of the chapters (and rebuild the app), run `yarn build`. This will `rm -rf` anything previously generated in the `/dist` directory.
+* To generate a single chapter without removing all the previously generated content, run `yarn generate src/chapters/00-your-chapter-to-edit-here.md`. This will rebuild _just_ the chapter that is specified without deleting everything else in the `/dist` directory.
+
+### Code changes
+
+To make code changes, you will first need to locate the chapter you wish to edit in `/src/chapters`. Depending on the type of code change you'd like to make, you will need to use different commands.
+
+#### Running commands
+
+If you'd like to run a command, you will need to use the `run: command`. Be sure to include `cwd=super-rentals` so that the file is generated in the app's directory.
+
+For example:
+
+    ```run:command hidden=true cwd=super-rentals
+    ember generate component rental
+    ```
+
```

Comment:
Mention that the output is included in the final markdown, if this is not what you want you can use `hidden=true` or `captureOutput=false` as appropriate. (Link to README maybe)